src/content/subgames.cpp

```diff
@@ -134,8 +134,50 @@ SubgameSpec findSubgame(const std::string &id)
 		}
 	}
 
-	if (game_path.empty())
-		return SubgameSpec();
+	std::string idv = id;
+
+	// Try to find aliased game
```

Comment:
Is it necessary to make this a separate condition? Would there be any drawbacks to always assuming the game may be aliased at the beginning of the search?

Reply:
Well, it should try to find a game proper going by that ID, and only failing that look through the aliases.

Reply:
Is that because two games A and B could be configured such that game B has the name of game A as one of its aliases, and the search should always choose game A if games A and B are both installed?

Reply:
Yes